packaging/rpm/README.md

```diff
@@ -0,0 +1,40 @@
+# Building a .rpm package of the CLI
+
+The RPM package of the CLI is simply a metapackage that installs the CLI gem directly through Ruby. Therefore, 
+the package itself only serves to facilitate the installation and inclusion of non-ruby dependencies.
+
+## Requirements
+
+The `gem2rpm` gem can be used to manage the package templates, even though it's not necessarily used to build the 
+package itself. To install it, simply run:
+
+```
+gem install gem2rpm 
+```
+
+To build the actual packages, you'll also need the `rpm` package from brew:
+
+```
+brew install rpm
```

Comment:
This would only be needed if building on macOS?

Reply:
Well, that's a good point. Both debian and rpm use fairly 'standard' apps (so much so that they work just fine on Mac OS), so we aren't tied to building on Mac only. That being said, the act of building and releasing packages should only be handled by the code owners so it's not very likely that we'll use other OSs for this. I'll update this file accordingly to include 'native' package installation instructions as well.